Admin/dataobjects/AdminUser.php

```diff
@@ -171,8 +184,8 @@ public function isAuthenticated(AdminApplication $app)
 			$authenticated = true;
 		}
 
-		if ($this->force_change_password)
-			$authenticated = false;
+		$authenticated = ($authenticated && $this->isActive());
+		$authenticated = ($authenticated && !$this->force_change_password);
```

Comment:
Ease of readability. I agree it could be one statement and still be easy to read.